orion/controllers/announce/announceController.go

```diff
@@ -0,0 +1,115 @@
+package announce
+
+import (
+	"errors"
+	"net/http"
+	"regexp"
+	
+	"github.com/gin-gonic/gin"
+)
+
+const REGEX_ANNOUNCE_ID = `^[[:alnum:]]+(_[[:alnum:]]+)*$`
```

Comment:
Thank you for leaving these checks. `AnnounceId` won't be used. Instead we will just be using the Row `Id`